src/cmaps/index.js

```diff
@@ -1,5 +1,5 @@
-export { default as _itksnap } from "./_itksnap.json";
-export { default as _slicer3d } from "./_slicer3d.json";
+export { default as itksnap } from "./_itksnap.json";
+export { default as slicer3d } from "./_slicer3d.json";
```

Comment:
Please note that the underscore prefix for _itksnap and _slicer3d denote that these are indexed colormaps for drawing pens, to distinguish them from interpolated/continuous colormaps used for scalar voxel data. If you change this, some demos will list poorly suited color schemes, e.g. 
  https://niivue.github.io/niivue/features/colormaps.html
I am fine if you want to use a different method of discriminating these (e.g. a ‘cmaps’ folder for colormaps and ‘maps’ for drawing colors), but you will need to update the demos and my standalone software.